worker-document/src/main/java/com/hpe/caf/worker/document/BulkDocumentMessageProcessor.java

```diff
@@ -112,12 +112,14 @@ public void processTasks() throws InterruptedException
 
                 documentWorkerTask.raiseAfterProcessTaskEvent();
 
+                // Unload the scripts
+                documentWorkerTask.unloadScripts();
+
```

Comment:
You need to think out the exception handling in this bulk worker module also.  Exceptions thrown when loading the scripts, or when executing some of their pre-processing handlers (i.e. `onProcessTask()`, `onBeforeProcessDocument()`, etc.), or transient or other exceptions from the worker processing, or exceptions from the post-processing handlers (i.e. `onAfterProcessDocument()`, `onAfterProcessTask()`) would all cause the memory leak to occur.  This only fixes it on the happy path.